content/intro-to-storybook/vue/en/data.md

```diff
@@ -214,21 +196,20 @@ export const Empty = () => ({
 Similarly, we need to use `PureTaskList` in our Jest test:
 
 ```js
-
-//tests/unit/TaskList.spec.js
+// src/components/PureTaskList.spec.js
 import Vue from 'vue';
-import PureTaskList from '../../src/components/PureTaskList.vue';
-import { withPinnedTasksData } from '../../src/components/PureTaskList.stories';
+import { WithPinnedTasks } from './PureTaskList.stories';
 
 it('renders pinned tasks at the start of the list', () => {
-  const Constructor = Vue.extend(PureTaskList);
+  // render WithPinnedTasks story
+  const Constructor = Vue.extend(WithPinnedTasks);
   const vm = new Constructor({
-    propsData: { tasks: withPinnedTasksData },
+    propsData: WithPinnedTasks.args,
   }).$mount();
-  const lastTaskInput = vm.$el.querySelector('.list-item:nth-child(1).TASK_PINNED');
+  const firstTaskPinned = vm.$el.querySelector('.list-item:nth-child(1).TASK_PINNED');
 
   // We expect the pinned task to be rendered first, not at the end
-  expect(lastTaskInput).not.toBe(null);
+  expect(firstTaskPinned).not.toBe(null);
```

Comment:
@andre-brdoch see my previous comment regarding the test and associated [issue](https://github.com/andre-brdoch/learnstorybook-v6-vue/issues/3) for more details. 

Let me know when you've made the changes and we'll get this one merged.